crates/cast/tests/cli/zk.rs

```diff
@@ -511,3 +512,39 @@ casttest!(test_zk_cast_custom_signature, async |prj, cmd| {
         .get_output()
         .stdout_lossy();
 });
+
+casttest!(test_zk_cast_call_with_trace, async |_prj, cmd| {
+    let node = ZkSyncNode::start().await;
+    let url = node.url();
+
+    let (_, private_key) = ZkSyncNode::rich_wallets()
+        .next()
+        .map(|(addr, pk, _)| (addr, pk))
+        .expect("No rich wallets available");
+
+    // Deploy counter
+    cmd.cast_fuse()
+        .args(["rpc", "hardhat_setCode", COUNTER_ADDRESS, COUNTER_BYTECODE, "--rpc-url", &url])
+        .assert_success();
+
+    cmd.cast_fuse()
+        .args([
+            "call",
+            "--trace",
+            COUNTER_ADDRESS,
+            "number()",
+            "--rpc-url",
+            &url,
+            "--private-key",
+            private_key,
+            "--zksync",
+        ])
+        .assert_success()
+        .get_output()
+        .stdout_lossy()
+        .contains(
+            "Traces:
+  [43369] 0x70997970C51812dc3A010C7d01b50e0d17dc79C8::number()
```

Comment:
```suggestion
  [43369] {{COUNTER_ADDRESS}}::number()
```
Or, do like foundry does with the `assert_eq` methods and put a `[..]` thing here (forgot the name)

Reply:
(the code in the suggestion is pseudocode coz there's still a `format!` missing)